README.md

```diff
@@ -67,68 +58,8 @@ You will need at least the upstream CAPI components, the Talos bootstrap provide
 
 ## Usage
 
-CACPPT supports a single API type, a TalosControlPlane.
-You can create YAML definitions of a TalosControlPlane and `kubectl apply` them as part of a larger CAPI cluster deployment.
-Below is a bare-minimum example.
-
-A basic config:
-
-```yaml
-apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
-kind: TalosControlPlane
-metadata:
-  name: talos-cp
-spec:
-  version: v1.18.1
-  replicas: 1
-  infrastructureTemplate:
-    kind: MetalMachineTemplate
-    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-    name: talos-cp
-  controlPlaneConfig:
-    init:
-      generateType: init
-    controlplane:
-      generateType: controlplane
-```
-
-Note you must provide an infrastructure template for your control plane.
-See your infrastructure provider for how to craft that.
-
-Note the generateType mentioned above.
-This is a required value in the spec for both init and controlplane nodes.
-For a no-frills control plane config, you can simply specify `controlplane` depending on each config section.
-When creating a TalosControlPlane this way, you can then retrieve the talosconfig file that allows for osctl interaction with your nodes by doing something like `kubectl get talosconfig -o yaml talos-cp-xxxx -o jsonpath='{.status.talosConfig}'` after creation.
-
-If you wish to do something more complex, we allow for the ability to supply an entire Talos config file to the resource.
-This can be done by setting the generateType to `none` and specifying a `data` field.
-This config file can be generated with `talosctl config generate` and the edited to supply the various options you may desire.
-This full config is blindly copied from the `data` section of the spec and presented under `.status.controlPlaneData` so that the upstream CAPI controllers can see it and make use.
-
-An example of a more complex config:
-
-```yaml
-apiVersion: control-plane.cluster.x-k8s.io/v1alpha2
-kind: TalosControlPlane
-metadata:
-  name: talos-0
-  labels:
-    cluster.x-k8s.io/cluster-name: talos
-spec:
-  controlPlaneConfig:
-    init:
-        generateType: none
-        data: |
-            version: v1alpha1
-            machine:
-            type: init
-            token: xxxxxx
-            ...
-            ...
-            ...
-  ...
-  ...
-```
+You can use recommended [Cluster API templates](https://github.com/talos-systems/cluster-api-templates) provided by Sidero Labs.
```

Comment:
is it fair to remove all docs here? should we still keep a reference of available options?

Reply:
Brought these back in and reworked the flow of that section a bit.